keyboards/1upkeyboards/sweet16/keymaps/ridingintraffic/config.h

```diff
@@ -0,0 +1,19 @@
+#ifndef CONFIG_RIDINGINTRAFFICTD
+#define CONFIG_RIDINGINTRAFFICTD
+
+/* tap dance stuff*/
+#include "../../config.h"
```

Comment:
This line is not actually needed, and should be removed. 

```suggestion
```

Specifically, the build script automatically includes all of the config.h files, so this is not necessary.